### PR TITLE
Add Dockerfile in order to install and run lpy

### LINKED ISSUE #9: Create a Dockerfile for Lpy
close because duplicate of #1

### PR DESCRIPTION
**Pull request by naittaleb**
close #9

### CODE REVIEW

**Comment by jlegrand62** [comment]
I had to rebase this branch with ``

Documentation here: https://docs.romi-project.eu/Scanner/docker/virtualplantscanner_docker/

IF YOU MAKE SOME CHANGES UPDATE THE DOCUMENTATION!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+COPY romiscanner.yaml romiscanner.yaml
+
+RUN conda env create -f romiscanner.yaml && \
+    conda clean -a -y
+
+ARG ROMISCAN_BRANCH=dev
+ARG ROMIDATA_BRANCH=dev
+ARG ROMISCANNER_BRANCH=master
+
+RUN git clone --branch $ROMISCAN_BRANCH https://github.com/romi/romiscan && \
```
Comment (naittaleb) [inline docker/Dockerfile:48]: @jlegrand62  why do we need romiscan?

Comment (jlegrand62) [reply docker/Dockerfile:48]: how do you start a task without `romiscan` ?

**Comment by naittaleb** [comment]
duplicate of #8

**Pull request closed without merging**